hummingbird/ml/supported.py

```diff
@@ -24,6 +24,7 @@
 GaussianNB,
 GradientBoostingClassifier,
 GradientBoostingRegressor,
+GridSearchCV,
```

Comment:
Can you please also add `RandomizedSearchCV` here?

Reply:
done.